TipCatDotNet.Api/Models/HospitalityFacilities/Validators/IdentityHashValidator.cs

```diff
@@ -0,0 +1,30 @@
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using FluentValidation;
+using TipCatDotNet.Api.Data;
+using Microsoft.EntityFrameworkCore;
+
+namespace TipCatDotNet.Api.Models.HospitalityFacilities.Validators
+{
+    public class IdentityHashValidator : AbstractValidator<string?>
```

Comment:
Don't think we need this one. The check looks like a part of general application flow